test/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -28,18 +28,21 @@ envoy_cc_test(
 envoy_cc_test(
     name = "quic_platform_test",
     srcs = ["quic_platform_test.cc"],
+    copts = ["-Wno-unused-parameter"],
```

Comment:
nit: Add a quiche_copt for it?

Reply:
This might be only place need copts in this BUILD file. If it turns out that we need it in more places, I would prefer to define quiche_copt in a centralize place instead of defining it repeatedly in each BUILD file.

Reply:
SGTM.